src/Orc.Controls/Controls/OpenFilePicker/Views/OpenFilePicker.xaml

```diff
@@ -20,26 +20,29 @@
 
         <Label Grid.Column="0" 
                Content="{Binding LabelText}"
-               Width="{Binding LabelWidth}" />
+               Width="{Binding LabelWidth}" 
+               VerticalAlignment="Center"
+               Margin="0"/>
 
         <TextBox AutomationProperties.AutomationId="{x:Static automation:OpenFilePickerMap.SelectedFileTextBoxId}"
                  Grid.Column="1" 
                  IsReadOnly="True" 
                  Text="{Binding SelectedFileDisplayName, Mode=OneWay}" 
-                 Height="{Binding ElementName=OpenFileButton, Path=ActualHeight}"/>
+                 Height="{Binding ElementName=OpenFileButton, Path=ActualHeight}"
```

Comment:
Please remove it (and also **remove it from other pickers**, if they have it too) Binding to sizes 95% in cases is wrong solution. I understand that maybe it's not your code, but as long as we noticed it - we should get rid of it: please try to remove it or use VerticalAlignment=Stretch -> whatever will work